app/views/users/show.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1>My events</h1>
+<p>Find me in app/views/users/show.html.erb</p>
+
+<%= @user.username %>
+<br>
+Hosted Events:<br>
+<% @user.events.each do |e| %>
+  <%= e.title%><br>
+<% end %>
+<br>
+
+Attended Events:<br>
+<br>
+Past Events:<br>
+<% @user.attended_events.each do |e| %>
+  <% if event_past?(e.date)%>
+    <%= e.title%><br>
+  <% end %>
+<% end %>
+
+<br>
+
+Upcoming Events:<br>
+<% @user.attended_events.each do |e| %>
+  <% unless event_past?(e.date)%>
+    <%= e.title%><br>
+  <% end %>
+<% end %>
```

Comment:
Consider using a partial here. Since you are rendering an event each time in each case. You should create an event partial, then render any event collection like this `render @events` or `render @user.events` :wink: